src/config/sharp.ts

```diff
@@ -11,30 +11,30 @@ export default {
         smartSubsample: true,
       },
     },
-    jpeg: {
-      ext: ".jpg",
-      outputOptions: {
-        chromaSubsampling: "4:4:4",
-        optimiseCoding: true,
-        quantisationTable: 3,
-        progressive: false,
-        optimiseScans: false, // required mozjpeg support
-        trellisQuantisation: true, // required mozjpeg support
-        overshootDeringing: true, // required mozjpeg support
-      },
-    },
-    png: {
-      ext: ".png",
-      outputOptions: {
-        chromaSubsampling: "4:4:4",
-        optimiseCoding: true,
-        quantisationTable: 3,
-        progressive: false,
-        optimiseScans: false, // required mozjpeg support
-        trellisQuantisation: true, // required mozjpeg support
-        overshootDeringing: true, // required mozjpeg support
-      },
-    },
+    // jpeg: {
```

Comment:
So I'm getting rid of these because they're not getting used in the UI, and I can't quite imagine why they'd get used, and we can always generate them in the future?